CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ FetchContent_Declare(inja URL https://github.com/pantor/inja/archive/refs/tags/v
 FetchContent_MakeAvailable(inja)
 
 add_library(ysmm-cl SHARED
-    src/cl.cpp
+    src/libysmm.cpp
```

Comment:
Might be worth renaming to `libysmm_cl.cpp` instead, as the longer term plan is to add a level zero backend in addition to OpenCL.

Reply:
done and PR updated